frontend/src/app/main/ui/inspect/right_sidebar.scss

```diff
@@ -153,14 +154,6 @@
   --tabs-nav-padding-inline-start: 0;
   --tabs-nav-padding-inline-end: var(--sp-m);
 
-  /* same height as .element-options in workspace/sidebar/options.scss */
-  /* which is one of the parents of this component */
-  --max-inspect-tab-height: var(--sidebar-element-options-height);
-  max-block-size: var(--max-inspect-tab-height);
-
+  block-size: calc(100vh - px2rem(200)); // TODO: Fix this hardcoded value
```

Comment:
This is a hardcoded value used in other tabs in the inspect tab. Not sure why it is done this way.

Reply:
This code is working only on the viewer sidebar, not in the inspect tab.  Forget the error of the scrollbar is not happening now.